nixos/modules/system/boot/clevis.md

```diff
@@ -0,0 +1,49 @@
+# Clevis {#module-boot-clevis}
+
+[Clevis](https://github.com/latchset/clevis)
+is a framework for automated decryption policy of ressources.
+Clevis allows for secure unattended disk decryption during boot, using decryption policies that must be satisfied for the data to decrypt.
+
+
+## Create a secret {#module-boot-clevis-create-secret}
+
+Secrets have to be created through the clevis command line. 3 types of policies are supported:
+
+1) TPM policies
+
+Secrets are binded against the presence of a TPM2 device, for example:
+```
+echo hi | clevis encrypt tpm2 '{}' > hi.jwe
```

Comment:
This file is encrypted and therefore safe to store in the Nix store. Is that right? Would appreciate a line in the documentation confirming that's the case.